### ISSUE TITLE
Multiple Targets: No Arrows

### ISSUE THREAD

**Issue by Amorik** [nbs-system/mapster#7]
Hello,

I've been playing with this plugin for a little bit. It seems that when using multiple targets arrows are no longer displayed between the origins and the targets. Is this intentional?

Thanks

~Regards

**Comment by xarkes** [comment]
Hi,
This is not intentional, could you please add a screenshot and check your javascripts logs ? (And paste if any warning/error)
Thanks

**Comment by Amorik** [comment]
Hi,

Screenshot below. Not sure what logs to check specifically, but i do not see anything in the kibana.log file. On that note, I've just realized I have posted in the wrong repository. I should move this to elastickent fork as I'm using his version for use with 5.4 of Kibana/ES. My apologies.

![image](https://cloud.githubusercontent.com/assets/23191740/26552099/a9aa373a-4453-11e7-92af-994ace6076cc.png)

**Comment by xarkes** [comment]
His version will be merged soon or later so I don't think there is any problem posting your issue here.
I meant javascript logs. Just open the map and right clic -> inspect in your web browser, then in the javascript console you should see some warnings/errors.

**Comment by Amorik** [comment]
Not sure how to copy and paste from that console atm. Below is the screenshot of the console.

There is only one error message that repeats as the map refreshes with new events: "TypeError: Cannot read property 'lat' of undefined".

My ES cluster has no security on it and the query is restricted to only match entries that have a geo-location  set. If its not ES then its possible that my target geohash might be causing the problem; it is as follows:
{"loc1":[45.416667,-75.7], "loc2":[37.771,-122.41]}

perhaps this is related to issue #6?

![image](https://cloud.githubusercontent.com/assets/23191740/26559694/3c1708e8-447f-11e7-8c07-80426f0ca774.png)

**Comment by xarkes** [comment]
Yes looks like related to the issue. Try to follow the indications in my last comment on this issue

**Comment by jvoisin** [comment]
@Amorik is this fixed on your side?
